modules/distr-tracing-otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -35,5 +63,6 @@ spec:
     - action: labeldrop
       regex: job
 ----
-<1> The name of the internal metrics port for the OpenTelemetry Collector. This port name is always `metrics`.
-<2> The name of the Prometheus exporter port for the OpenTelemetry Collector. This port name is defined in the `.spec.ports` section of the `OpenTelemetryCollector` CR.
+<1> The name of the OpenTelemetry customer resource e.g. `<cr-name>-collector`.
```

Comment:
I suggest moving the `<cr-name>-collector` example into the code block. See my previous suggestion.

```suggestion
<1> The name of the OpenTelemetry custom resource.
```